.travis/run.sh

```diff
@@ -9,22 +9,14 @@ if [[ "$(uname -s)" == "Darwin" ]]; then
     PATH="$PYENV_ROOT/bin:$PATH"
     eval "$(pyenv init -)"
 
-    if [[ "${OPENSSL}" != "0.9.8" ]]; then
-        # set our flags to use homebrew openssl
-        export ARCHFLAGS="-arch x86_64"
-        # if the build is static we need different LDFLAGS
-        if [[ "${CRYPTOGRAPHY_OSX_NO_LINK_FLAGS}" == "1" ]]; then
-            export LDFLAGS="/usr/local/opt/openssl/lib/libssl.a /usr/local/opt/openssl/lib/libcrypto.a"
-        else
-            export LDFLAGS="-L/usr/local/opt/openssl/lib"
-        fi
-        export CFLAGS="-I/usr/local/opt/openssl/include"
-        # The Travis OS X jobs are run for two versions
-        # of OpenSSL, but we only need to run the
-        # CommonCrypto backend tests once. Exclude
-        # CommonCrypto when we test against brew OpenSSL
-        export TOX_FLAGS="--backend=openssl"
+    # set our flags to use homebrew openssl
+    # if the build is static we need different LDFLAGS
+    if [[ "${CRYPTOGRAPHY_OSX_NO_LINK_FLAGS}" == "1" ]]; then
+        export LDFLAGS="/usr/local/opt/openssl/lib/libssl.a /usr/local/opt/openssl/lib/libcrypto.a"
+    else
+        export LDFLAGS="-L/usr/local/opt/openssl/lib"
     fi
+    export CFLAGS="-I/usr/local/opt/openssl/include"
```

Comment:
That's fine. I suspected there were good reasons for these, I just didn't know what they were. In that case, this LGTM, but I can't merge so let's wait until someone with big green button privileges comes along.